.github/workflows/triage.yml

```
@@ -67,6 +67,7 @@ jobs:
             - label: missing description
               path: Casks/.+
               missing_content: \n  desc .+\n
+              missing_content: cask "font-
```

YAML mapping (dictionary/hash) would mean this is a duplicate key. Depending on YAML implementation, this is either an error or only one of the conditions would be retained.

Probably want to make use of a YAML sequence (list/array) or use an `|` regex.
```suggestion
              missing_content:
                - \n  desc .+\n
                - cask "font-
```

https://github.com/Homebrew/actions/blob/master/label-pull-requests/main.mjs#L272-L280
```mjs
        if (Array.isArray(constraint.missing_content)) {
            for (const content of constraint.missing_content) {
                if (file.content.match(content)) {
                    return false
                }
            }
        } else if (file.content.match(constraint.missing_content)) {
            return false
        }
```